### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by lexiwzx** [sherleens/WSCNet#1]
May I ask when the code will be available? Or will the trained model be provided?

Thanks.

**Comment by sherleens** [comment]
Yes, we will release the final version as well as the model ASAP.

**Comment by ccivit** [comment]
I'm dropping in to also ask if there's an estimated timeline for when the models will be released. It would be helpful to reproduce the results.

Thanks,

**Comment by DinghaoXi** [comment]
I hope to see the released model too.
